native_client/javascript/Makefile

```diff
@@ -29,21 +32,28 @@ package.json: package.json.in
 		-e 's/$$(PROJECT_VERSION)/$(PROJECT_VERSION)/' \
 		package.json.in > package.json && cat package.json
 
-configure: deepspeech_wrap.cxx package.json
+npm-dev: package.json
+	# node-gyp@5.x behaves erratically with VS2015 and MSBuild.exe detection
+ifeq ($(TARGET),host-win)
+	npm install node-gyp@4.x
+endif
+	npm --no-color install --prefix=$(NPM_ROOT)/../ --ignore-scripts --force --verbose --production=false .
+
+configure: deepspeech_wrap.cxx package.json npm-dev
 	$(NODE_BUILD_TOOL) configure $(NODE_BUILD_VERBOSE)
 
 build: configure deepspeech_wrap.cxx
```

Comment:
> This target also depends on `node-pre-gyp`, shouldn't the installation of that tool be a dep of this too?

`node-pre-gyp` is installed when we run the `npm-dev` target, since it's a `devDependencies`.

Reply:
Yes, sorry, I made this comment while looking at an older version of the code.